src/main/kotlin/io/github/lagersystembackend/product/Product.kt

```diff
@@ -38,6 +38,14 @@ data class AddProductNetworkRequest(
     val spaceId: String
 )
 
+@Serializable
+data class UpdateProductNetworkRequest(
+    val id: String,
+    val name: String? = null,
+    val description: String? = null,
+    val spaceId: String? = null
```

Comment:
Soll die _spaceId_ bei einem Update veränderbar sein? In dem Fall wäre die move-Methode mehr oder weniger überflüssig.